spyder/preferences/general.py

```diff
@@ -246,9 +277,15 @@ def setup_page(self):
 
         screen_resolution_layout.addLayout(screen_resolution_inner_layout)
         screen_resolution_group.setLayout(screen_resolution_layout)
+        if sys.platform == "darwin":
+            interface_tab = self.create_tab(screen_resolution_group,
+                                            interface_group, macOS_group)
+        else:
+            interface_tab = self.create_tab(screen_resolution_group,
+                                            interface_group)
 
         tabs = QTabWidget()
-        tabs.addTab(self.create_tab(screen_resolution_group, interface_group),
+        tabs.addTab(interface_tab,
                     _("Interface"))
```

Comment:
Please change the previous two lines to be

    tabs.addTab(interface_tab, _("Interface"))